README.md

```diff
@@ -135,11 +135,17 @@ Options:
                                   hg|\.mypy_cache|\.nox|\.tox|\.venv|\.svn|_bu
                                   ild|buck-out|build|dist)/]
 
+  --extend-exclude TEXT           Like --exclude, but adds additional files
```

Comment:
This comment has nothing to do with this line, but I can't comment on the line I want to due to GitHub Review UI limitations :(

https://github.com/psf/black/pull/2005/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R160

This explanation is already not complete, notably `using configuration from [PATH]` lines are enabled by this flag, but ignoring that, I think just stating "due to exclusion configuration" or "due to exclusion regexes" would be an improvement.

*don't forget update the docs/installation_and_usage.md document too if you decide this is okay

Reply:
I like the suggestion, and went with "exclusion patterns" (as it slightly mirrors the wording `pyproject.toml` doc).
And speaking of, I edited that doc to mention the other exclusion options as well (by name)